soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2082,6 +2082,8 @@ void Player_ProcessControlStick(PlayState* play, Player* this) {
         direction = (u16)((s16)(sControlStickWorldYaw - this->actor.shape.rot.y) + 0x2000) >> 14;
     }
 
+    GameInteractor_ExecuteOnPlayerProcessStick();
```

Comment:
It feels a bit odd having this *in the middle* of `Player_ProcessControlStick` - generally we try to follow a "before" or "after" pattern so it's easy for people using the hooks to know what to expect when hooking in. i don't think this is a blocker, but i'd love to hear any thoughts on what it would take to move this up to the top of this function.

Reply:
FYI I've waited with merging until we have an answer if this can easily be moved to the beginning or end of the function. If not, that's fine too as it's not a huge deal breaker, just want to be sure about it.

Reply:
i think passing the values into the hook and having it be a "modify" of some sort might make sense. it could even be a VB should because it would happen if we "should modify processed control stick values"

Reply:
For now, I like keeping it as it is since there's no current use for the values, but it can easily be changed to that if the need arises.

Reply:
Sounds good to me